docs/DeveloperGuide.adoc

```diff
@@ -540,11 +540,36 @@ _{More to be added}_
 [appendix]
 == Glossary
 
-[[mainstream-os]] Mainstream OS::
-Windows, Linux, Unix, OS-X
```

Comment:
Hello @liawsy :) so sorry but is it possible to help me add this part in as well? it's for a part in NFRs!